### PR TITLE
ARROW-118 Release 0.5.1

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: bindings/python/docs/source/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Changes in Version 0.5.1
+------------------------
+- Fixed auto-discovery of schemas for aggregation and ``numpy`` methods.
+- Added Documentation for Auto-Discovery of Schemas.
```
[the remark this answers is not in the document]
Comment (blink1073) [reply bindings/python/docs/source/changelog.rst]: Fixed

**Review by ShaneHarvey — APPROVED** [review]
LGTM but can you merge https://github.com/mongodb-labs/mongo-arrow/pull/81 first?

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]
We need to update the version too: https://github.com/mongodb-labs/mongo-arrow/blob/main/bindings/python/pymongoarrow/version.py#L15

**Comment by blink1073** [comment]
I was planning to handle versions as part of the release process, since I have to bump to dev version as well.

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**